titus-server-gateway/src/main/java/com/netflix/titus/gateway/service/v3/internal/ExtendedJobSanitizer.java

```diff
@@ -151,6 +157,26 @@ public ExtendedJobSanitizer(JobManagerConfiguration jobManagerConfiguration,
                 }
             });
 
+            Map<String, String> defaultContainerAttributes = new HashMap<>();
+            violations.findViolation(CONTAINER_ACCOUNT_ID_REQUIRED_FEATURE).ifPresent(nonCompliance -> {
+                if (!StringExt.isEmpty(jobManagerConfiguration.getDefaultContainerAccountId())) {
+                    defaultContainerAttributes.put(JobAttributes.JOB_CONTAINER_ATTRIBUTE_ACCOUNT_ID, jobManagerConfiguration.getDefaultContainerAccountId());
+                }
+            });
+            violations.findViolation(SUBNETS_REQUIRED_FEATURE).ifPresent(nonCompliance -> {
```

Comment:
What is a user sets an account id (non-default), but not subnets?  Setting default subnets will be incorrect.
I think we should validate/set this data together, as they must be mutually consistent.